src/appleseed/foundation/platform/path.cpp

```diff
@@ -47,13 +47,9 @@
 // Platform headers.
 #if defined __APPLE__
 #include <mach-o/dyld.h>
-#elif defined __linux__
-#include <pwd.h>
+#elif defined __linux__ || defined __FreeBSD__
 #include <sys/types.h>
-#include <unistd.h>
-#elif defined __FreeBSD__
 #include <pwd.h>
-#include <sys/sysctl.h>
```

Comment:
Yes, I know it's tempting to move all Linux/FreeBSD includes under one hood, but right now Linux code path does not use `sys/sysctl.h` header: its `get_executable_path()` implementation reads from `/proc/self/exe`.

Technically it does not hurt anything, because `sys/sysctl.h` exists on both systems, but it's not very neat.  (Let me think if I can improve Linux implementation while here: reading from `procfs` involves VFS layer, perhaps there is a more direct way to ask kernel for executable path, like we do for FreeBSD.)